frontend/src/components/LineChart/LineChart.js

```diff
@@ -61,9 +60,13 @@ const LineChart = ({ data, title, xAxis, yAxis }) => {
             }
           />
         }>
-        <VictoryAxis tickFormat={handleXTickFormat} />
-        <VictoryAxis dependentAxis domain={domain} tickFormat={handleYTickFormat} />
-        <VictoryLine data={data} />
+        <VictoryAxis fixLabelOverlap tickFormat={handleXTickFormat} />
+        <VictoryAxis dependentAxis tickFormat={handleYTickFormat} />
+        <VictoryStack colorScale={STACK_COLORS}>
```

Comment:
`<LineChart>` isn't a stack. Each line should be drawn independently from the other. For instance, if I alter the layout for "Business Counts" to be a line chart, this point with a value of 0 should be down at the bottom and not up around 800.

<img width="589" alt="image" src="https://user-images.githubusercontent.com/244952/165124645-570a4396-7f28-43f8-a1c4-2957b4ba00c2.png">